### PR TITLE
add DNS troubleshooting documentation

### PR DESCRIPTION
**Pull request by arthurwolf**
## Checklist

 - [ ] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [ ] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: docs/src/how-to/install/troubleshooting.md [hunk 1]
```diff
@@ -263,3 +263,154 @@ p: the expected ping (how many pings have not returned)
...
+
+### Testing public domains.
+
+From your own computer (not from the Wire backend), test that you can reach all sub-domains you setup during the Wire installation:
+
+* `assets.youdomain.com`
+* `teams.yourdomain.com`
+* `webapp.yourdomain.com`
+* `accounts.yourdomain.com`
+* etc...
```
Comment (julialongtin) [inline docs/src/how-to/install/troubleshooting.md:281]: add the rest of the domains, not etc...

File: docs/src/how-to/install/troubleshooting.md [hunk 2]
```diff
@@ -263,3 +263,154 @@ p: the expected ping (how many pings have not returned)
...
+And if the domain can not be resolved, it will be something like this:
+
+```
+Server:         127.0.0.53
+Address:        127.0.0.53#53
+
+** server can't find minio-external: NXDOMAIN
+```
+
+If you can not reach some of the domains from the SNS pod, you need to try those from one of the servers running kubernetes (kubernetes host):
```
Comment (julialongtin) [inline docs/src/how-to/install/troubleshooting.md:361]: get the region from the values.yaml file.

File: docs/src/how-to/install/troubleshooting.md (hunk 1 above)
Comment (arthurwolf) [reply docs/src/how-to/install/troubleshooting.md]: corrected in latest commit.

File: docs/src/how-to/install/troubleshooting.md (hunk 2 above)
Comment (arthurwolf) [reply docs/src/how-to/install/troubleshooting.md]: done.

**Review by julialongtin — APPROVED** [review]

**Pull request merged**